### DISCUSSION TITLE
struct wrapping options

### DISCUSSION THREAD

**Discussion by ingDIY** [uncrustify/uncrustify#4331] [Q&A]
Hello, 
I'm trying to have the structs initialized one element per line. Unfortunatlry uncrustify removes the new lines like this:
![Immagine 2024-06-20 164815](https://github.com/uncrustify/uncrustify/assets/10012263/7261aa9f-b24f-42ae-87bc-8f5d4128c6b7)

What options should I check?
Thanks!

PS: i attach my current config file
[uncrustify_config.txt](https://github.com/user-attachments/files/15915815/uncrustify_config.txt)

**Comment by guy-maurel** [comment]
same
